src/main/java/org/springframework/cloud/dataflow/module/deployer/cloudfoundry/CloudFoundryModuleDeployerProperties.java

```diff
@@ -82,6 +82,16 @@
 	 */
 	private String buildpack = "https://github.com/cloudfoundry/java-buildpack.git#master";
 
+	/**
+	 * The amount of memory (MB) to allocate, if not overridden per-module.
+	 */
+	private int memory = 1024;
+
+	/**
+	 * The amount of disk space (MB) to allocate, if not overridden per-module.
+	 */
+	private int disk;
```

Comment:
Seems like there's no adverse reaction to passing zero, but to avoid any future surprises, I think it'd be better to simply pass `null` if the value is not a positive integer.